api/src/main/java/org/openmrs/ConceptReferenceTerm.java

```diff
@@ -16,28 +16,52 @@
 import org.hibernate.search.annotations.DocumentId;
 import org.hibernate.search.annotations.Field;
 
+import javax.persistence.AttributeOverride;
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+
 /**
  * A concept reference term is typically name for a concept by which it is referred in another
  * institution like ICD9, ICD10, SNOMED that keeps a concept dictionary or any other OpenMRS
  * implementation
  *
  * @since 1.9
  */
+@Entity
+@Table(name = "concept_reference_term")
+@AttributeOverride(name = "name", column = @Column(name = "name"))
```

Comment:
@alinmihaila99, what are you overriding here?

Reply:
I overrided the column "name" to allow null